add doc for experimental variable tidb_store_batch_size (#11562)
pingcap/docs: system-variables.md

- Unit: Seconds
- This variable is used to set the refresh time of [statement summary tables](/statement-summary-tables.md).

### `tidb_store_batch_size` <span class="version-mark">New in v6.5.0</span>

> **Warning:**
>
> Currently,`tidb_store_batch_size` is an experimental feature. It is not recommended that you use it in production environments.

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Type: Integer
- Default value: `0`
- Range: `[0, 25000]`

> - This variable is used to control the batch size of table reader Coprocessor Tasks of `IndexLookUp` executor. `0` means disable batch, you can increase it to optimize the extremely slow queries.

Reviewer: ```suggestion
- This variable is used to control the batch size of table reader Coprocessor Tasks of the `IndexLookUp` executor. `0` means disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.
```

- Unit: Seconds
- This variable is used to set the refresh time of [statement summary tables](/statement-summary-tables.md).

> ### `tidb_store_batch_size` <span class="version-mark">New in v6.5.0</span>

Reviewer 2: ```suggestion
### `tidb_store_batch_size`
```

> **Warning:**
>

> > Currently,`tidb_store_batch_size` is not stable yet. This variable may be removed in the future, it is not recommended that you use it in production environments.
> > Suggest to use the default value.

Reviewer 2: ```suggestion
> Currently, `tidb_store_batch_size` is not stable yet. This variable might be removed in a future release. It is not recommended that you use it in production environments. It is not recommended to change the default value.
```

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Type: Integer
- Default value: `0`
- Range: `[0, 25000]`

> - This variable is used to control the batch size of table reader Coprocessor Tasks of the `IndexLookUp` executor. `0` means disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.

Reviewer 2: ```suggestion
- This variable is used to control the batch size of the Coprocessor Tasks of the `IndexLookUp` operator. `0` means to disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.
```
